README.md

```diff
@@ -3,90 +3,107 @@
+<a href="https://docs.wandb.ai/guides/data-vis"><img src="./README_images/tables.png" width="13.5%" /></a>
+<a href="https://docs.wandb.ai/guides/sweeps"><img src="./README_images/sweeps.png" width="13.5%" /></a>
+<a href="https://docs.wandb.ai/guides/models"><img src="./README_images/models.png" width="13.5%" /></a>
+<a href="https://docs.wandb.ai/guides/launch"><img src="./README_images/launch.png" width="13.5%" /></a>
+</p>
 
-[Sign up for a free account →](https://wandb.com)
+Get started today with W&B, [sign up for a free account!](https://wandb.com)
 
-## Features
+<!-- | Experiments | Reports | Artifacts | Tables | Sweeps | Models | Launch |
+| ----------- | ------- | --------- | ------ | ------ | ------ | ------ |
+| Text        | Text    | Text      | Text   | Text   | Text   | Text   | -->
 
--   Store hyper-parameters used in a training run
--   Search, compare, and visualize training runs
--   Analyze system usage metrics alongside runs
--   Collaborate with team members
--   Replicate historic results
--   Run parameter sweeps
--   Keep records of experiments available forever
+&nbsp;
 
-[Documentation →](https://docs.wandb.com)
+# Documentation
 
-If you have any questions, please don't hesitate to ask in our [user forum](http://wandb.me/forum).
+See the [W&B Developer Guide](https://docs.wandb.ai/) and [API Reference Guide](https://docs.wandb.ai/ref) for a full technical description of the W&B platform. Here are some highlights:
 
-# 🤝 Simple integration with any framework
-Install `wandb` library and login:
-```
+&nbsp;
+
+# Install W&B
+
+Install the W&B SDK with [pip](https://pip.pypa.io/en/stable/):
+
+```bash
 pip install wandb
-wandb login
 ```
-Flexible integration for any Python script:
-```python
-import wandb
 
-# 1. Start a W&B run
-wandb.init(project="gpt3")
+&nbsp;
 
-# 2. Save model inputs and hyperparameters
-config = wandb.config
-config.learning_rate = 0.01
+# Quickstart
 
-# Model training code here ...
+Get started with W&B in three steps:
 
-# 3. Log metrics over time to visualize performance
-for i in range(10):
-    wandb.log({"loss": loss})
+1. First, sign up for a [free W&B account](https://wandb.ai/login).
+
+2. Next, log into W&B. Navigate to your terminal and type the following command:
+
+```python
+wandb.login()
 ```
 
-### [Try in a colab →](http://wandb.me/intro)
+3.5. Use the example code snippet below as a template to integrate W&B to your Python script:
 
-If you have any questions, please don't hesitate to ask in our [user forum](http://wandb.me/forum).
+```python
+import wandb
 
-![](https://i.imgur.com/TU34QFZ.png)
+# Start a W&B Run with wandb.init
+wandb.init(project="my_first_project")
 
-**[Explore a W&B dashboard](https://www.youtube.com/watch?v=gnD8BFuyVUA)**
+# Save model inputs and hyperparameters in a wandb.config object
+config = wandb.config
+config.learning_rate = 0.01
 
+# Model training code here ...
 
-# Academic Researchers
```

Comment:
I think we should keep this section (albeit with modifications -- mirroring and compressing https://wandb.ai/site/research) or at least highlighted that W&B is "Free for academic and open source projects."